website/docs/r/is_instance_group.html.markdown

```diff
@@ -71,9 +71,9 @@ The `ibm_is_instance_group` resource provides the following [Timeouts](https://w
 ## Argument reference
 Review the argument references that you can specify for your resource. 
 
-- `application_port` - (Optional, Integer) The instance group uses when scaling up instances to supply the port for the Load Balancer pool member.
-- `load_balancer` - (Optional, String) The load Balancer ID.
-- `load_balancer_pool` - (Optional, String) The load Balancer pool ID.
+- `application_port` - (Optional, Integer) The instance group uses when scaling up instances to supply the port for the Load Balancer pool member, when `application_port` specified `load_balancer` and `load_balancer_pool` mandatory. 
```

Comment:
Can you change description something like this
`application_port` - (Optional, Integer) The instance group uses when scaling up instances to supply the port for the Load Balancer pool member. The load_balancer and load_balancer_pool arguments must be specified when configured.